.github/workflows/ci.yml

```diff
@@ -133,7 +133,17 @@ jobs:
           echo ${{ steps.regression_labels.outputs.labels }}
           REGRESSION_LABELS=$(echo "${{ steps.regression_labels.outputs.labels }}" | awk -F, '{for (i=1; i<=NF; i++) if ($i ~ /^--/) print $i}' ORS=',' | sed 's/,$//')
           echo ${REGRESSION_LABELS}
-          tb env regression-tests coverage --wait $(echo ${REGRESSION_LABELS} | tr , ' ')
+
+          CONFIG_FILE=./tests/regression_config.yaml
```

Comment:
I'd simplify the name to regression.yaml, but just an opinion.

Reply:
Agree!